tests/cp_aks_zero/state.tf

```diff
@@ -0,0 +1,8 @@
+terraform {
+  backend "azurerm" {
+    resource_group_name  = "terraform-kubestack-testing"
```

Comment:
TODO: Update quickstart to also include `resource_group_name` and update instructions too.